### PR TITLE
Improve Jinja2 template

### PR DESCRIPTION
**Pull request by alexandreroutier**
When reviewing PR #62, I noticed a few mistakes in the generated template. I fixed this and took the oppportunity to update/harmonize it regarding the last modifications of Clinica.

### CODE REVIEW

File: clinica/pipelines/t1_volume_tissue_segmentation/t1_volume_tissue_segmentation_pipeline.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def build_input_node(self):
                                             self.bids_directory,
                                             T1W_NII)
         except ClinicaException as e:
-            err = 'Clinica faced error(s) while trying to read files in your CAPS directory.\n' + str(e)
+            err = 'Clinica faced error(s) while trying to read files in your BIDS directory.\n' + str(e)
```
Comment (mdiazmel) [inline clinica/pipelines/t1_volume_tissue_segmentation/t1_volume_tissue_segmentation_pipeline.py]: Just to confirm... this change isn't related to the goal of this PR, isn't ? (It seems like a forgotten typo from a previous PR)

Comment (alexandreroutier) [reply clinica/pipelines/t1_volume_tissue_segmentation/t1_volume_tissue_segmentation_pipeline.py]: When updating the Jinja2 using `t1-volume-tissue-segmentation` pipeline as a model, I noticed this typo. I forgot to make an atomtic commit.

**Comment by alexandreroutier** [comment]
> All seems ok.
> Only one thing, can you verify that the length of the lines in the template README.md.j2 are the right ones ?

What is the max length? I did not see anything on the [Developper Guide](https://gitlab.com/icm-institute/aramislab/clinica_aramis/-/wikis/docs/DeveloperGuide/BestPractices). Besides, I don't think we currently use a max length in the Markdown files for Clinica (see e.g. `docs/Pipelines/T1_FreeSurfer.md`).

**Comment by mdiazmel** [comment]
> What is the max length? I did not see anything on the Developper Guide. Besides, I don't think we currently use a max length in the Markdown files for Clinica (see e.g. docs/Pipelines/T1_FreeSurfer.md).

At some point, we used to follow the _[Google style guide](http://google.github.io/styleguide/pyguide.html)_ for Python. The reference is not anymore in the Developer's Guide (I'll update it).  The max length porposed by this guide is 80 characters.  

I've been trying to follow that for the code and also for the wiki (not for all the cases, but mostly). I think that is easier to read and to edit, particularly from a web editor.

**Comment by alexandreroutier** [comment]
Hi @mdiazmel,

Based on your PR #62 regarding `t1-linear`, I think I finish to complete the skeleton of a pipeline in Clinica.

I also try to 80-ish the Markdown file. URL in Markdown makes difficult to have 80 caracters in my opinion :thinking: A better solution would be to integrate an automatic parser/checker like https://remark.js.org :)

If you don't have any feedback anymore, you can safely merge this branch :)

Alex

**Comment by mdiazmel** [comment]
> I also try to 80-ish the Markdown file. URL in Markdown makes difficult to have 80 caracters in my opinion 🤔 A better solution would be to integrate an automatic parser/checker like https://remark.js.org :)

Yes, I think that we can omit the 80-characters policy for URLs!

I will merge this PR. Thanks @alexandreroutier .

**Pull request merged**